mlflow/server/js/src/common/components/IconButton.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Button } from 'antd';
+
+export class IconButton extends Component {
```

Comment:
A functional component should be enough?

Reply:
Yes. I'll fix.